lib/LogCollector.js

```diff
@@ -0,0 +1,231 @@
+/* Copyright @ 2016 Atlassian Pty Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+var Logger = require('./Logger.js');
+
+/**
+ * Stringifies given <tt>object</tt> without any circular references.
+ * @param {object} someObject the <tt>object</tt> to be stringified.
+ * @return {string} the result of <tt>JSON.stringify</tt> with the circular
+ * references skipped.
+ */
+function stringify(someObject){
+    const cache = [];
+    return JSON.stringify(someObject, function (key, value) {
```

Comment:
Ok, added another one: https://github.com/jitsi/jitsi-meet-logger/pull/1/commits/39570a32b191b3a02d738aaeaf2a090abb13a124 will remove the previous one with force push